flair/list.txt

```diff
@@ -114,6 +89,7 @@ activity.video-game
 activity.volleyball
 activity.wind-chime
 activity.wrapped-gift
+activity.xmas-hat
```

Comment:
error?

Reply:
I think it had been added in a previous commit and the list hadn't been updated